specification/containerregistry/data-plane/Microsoft.ContainerRegistry/preview/2019-08-15/containerregistry.json

```diff
@@ -2061,6 +2058,38 @@
         "writeEnabled": true
       }
     },
+    "RepositoryChangeableAttributes": {
+      "description": "Changeable attributes for Repository",
+      "properties": {
+        "deleteEnabled": {
+          "type": "boolean",
+          "description": "Delete enabled"
+        },
+        "writeEnabled": {
+          "type": "boolean",
+          "description": "Write enabled"
+        },
+        "listEnabled": {
+          "type": "boolean",
+          "description": "List enabled"
+        },
+        "readEnabled": {
+          "type": "boolean",
+          "description": "Read enabled"
+        },
+        "teleportEnabled": {
+          "type": "boolean",
+          "description": "Enables Teleport functionality on new images in the repository improving Container startup performance"
+        }
+      },
+      "example": {
```

Comment:
Nit: this seems redundant given that we have an example elsewhere.

Reply:
@jhendrixMSFT merging the PR since your comment is tagged "nit" :). Let me know if that's a concern